precompile/config_test.go

```diff
@@ -0,0 +1,63 @@
+// (c) 2022 Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package precompile
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ava-labs/subnet-evm/commontype"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVerifyPrecompileUpgrades(t *testing.T) {
```

Comment:
Can we add a similar test suite to verify that the `Equal` function works as expected for each of the different config types as well?

Reply:
done